PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -355,7 +371,7 @@
 ## - To be used in nanoAOD_customizeCommon() in nano_cff.py
 ###############################################################
 from PhysicsTools.PatAlgos.tools.jetTools import updateJetCollection
-def nanoAOD_addDeepInfoAK4CHS(process,addDeepBTag,addDeepFlavour,addParticleNet):
+def nanoAOD_addDeepInfoAK4CHS(process,addDeepBTag,addDeepFlavour,addParticleNet,addRobustParTAK4):
```

Comment:
This new option is clashing with the additions from the Tau POG: https://cmssdt.cern.ch/dxr/CMSSW/source/PhysicsTools/NanoAOD/python/nano_cff.py#171.

Would it make more sense to turn these args into kwargs and set defaults for all of them? I would leave addRobustPartAK4 set to False for now, we can revise this later.

Alternatively, just add the argument in the call I linked above (it might require a rebase to pull in the latest merged PRs)

Reply:
Right, we came to same conclusion (same goes for the backport by the tau pog, which has just been merged as well).
I'd personally also go for the kwargs variant with `addRobustParTAK4 = False` which would be the minimal invasive option.